examples/resources/versioning/bal_database.json

```diff
@@ -0,0 +1,38 @@
+{
+  "managementPolicy": {
+    "read": {
+      "default": {
+        "openassetio-mediacreation:managementPolicy.Managed": {},
+        "openassetio-mediacreation:content.LocatableContent": {}
+      }
+    }
+  },
+  "entities": {
+    "project_artwork/logos/openassetio": {
+      "versions": [
+        {
+          "traits": {
+            "openassetio-mediacreation:identity.DisplayName": {
+              "name": "The OpenAssetOI logo",
```

Comment:
[consider] 
```suggestion
              "name": "The OpenAssetIO logo",
```

Reply:
Was meant to be part of what was broken with the old version, but can conform if you think its better?

Reply:
Its fine if its intentional, but perhaps worth pointing out in the body of the text that version 2 fixes issues with version 1.

Reply:
Conformed [here](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/59/commits/6e66402e3046627a3d3d52751fff2797e14b5475 )